discovery_engine_core/web-api2/src/models.rs

```diff
@@ -126,48 +125,8 @@ impl AiDocument for PersonalizedDocumentData {
     }
 }
 
-/// Represents personalized documents query params.
-#[derive(Debug, Clone, Deserialize)]
-pub(crate) struct PersonalizedDocumentsQuery {
-    #[allow(dead_code)]
-    pub(crate) count: Option<usize>,
-}
```

Comment:
:grey_exclamation:  I moved all fully endpoint specific parts (query parameters, response body) to the endpoints, so that we don't have to jump between files to e.g. see which query parameters and endpoint accepts. This file still contains all models shared between various places.